### DISCUSSION TITLE
npm部署似乎只支持node15，不支持node16,17

### DISCUSSION THREAD

**Discussion by mefengl** [DIYgod/RSSHub#8823] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/71683364/148062049-8b6a7730-072b-4c0c-98b0-a7d27bc91e60.png)

上述教程，mac环境安装过程中，从node17试到node15，发现15能安装成功，其它会报错。

**Comment by mefengl** [comment, marked as answer]
我个人通过nvm安装不同的node 版本解决。

**Reply by mefengl** [reply]
即node v15

**Comment by shawn8888** [comment]
Windows下，安装 node-v16.13.2-x64.msi 正常运行

```
c:\>node -v
v16.13.2

```

**Reply by mefengl** [reply]
![image](https://user-images.githubusercontent.com/71683364/148978391-8f867f05-fa3d-494e-88ce-664ea1b9c831.png)
mac不行。
